### PR TITLE
added getIsTablet method to detect iPad or Android Tablet screen sizes

### PR DESCRIPTION
**Pull request by iotashan**
Handy method so you can switch layouts or styles for Tablets. Blatantly based on the Appcelerator Titanium / Alloy isTablet property code.

### CODE REVIEW

File: deviceinfo.js [hunk 1]
```diff
@@ -53,7 +56,4 @@ module.exports = {
   getDeviceCountry: function() {
     return RNDeviceInfo.deviceCountry;
   },
-  getTimezone: function() {
-    return RNDeviceInfo.timezone;
-  },
```
Comment (GantMan) [inline deviceinfo.js]: why are you removing timezone here?

**Comment by GantMan** [comment]
See comment above, also any new functionality needs to be reflected in the README, too.

**Comment by himanshusoni** [comment]
Can we please merge this PR
Thanks!

**Comment by GantMan** [comment]
PR has conflicts.  Someone will need to rebase.  Also they never updated.

**Comment by machour** [comment]
Thank you for this PR @iotashan. It seems that meanwhile isTablet() was already added in 0.9.7, so I'm closing this.

**Pull request closed without merging**